examples/unary-operators/unary_operators.md

```diff
@@ -0,0 +1,9 @@
+# Unary operators
+
+Ballerina supports unary operators that perform operations on a single operand. The unary `-` operator negates the value of a numeric type, effectively changing its sign. The unary `+` operator returns the value of the expression it is applied to, as long as that expression is a number. The bitwise NOT operator (`~`) inverts the bits of an integer, converting each 0 to 1 and each 1 to 0. 
```

Comment:
Still doesn't stick to spec terms.

- Avoid things like "effectively changing its sign"
- Avoid "as long as it is a number". If it is not a number, it isn't even allowed.

Reply:
I guess, We cannot use spec terms like `operands` in a ballerina by examples?

Reply:
Operand is a common term, IMO.

Reply:
yes, but for a `bbe` is it ok to use?

Reply:
updated